src/domains/citizen-auth-app/07_function_fast_login.tf

```diff
@@ -91,6 +92,140 @@ module "fast_login_snet" {
   }
 }
 
+## Create resources for fast-login on ITN Region
+
+module "fast_login_snet_itn" {
+  source                                    = "git::https://github.com/pagopa/terraform-azurerm-v3.git//subnet?ref=v8.22.0"
+  name                                      = format("%s-fast-login-snet-01", local.project_itn)
+  address_prefixes                          = var.cidr_subnet_fnfastlogin_itn
+  resource_group_name                       = data.azurerm_virtual_network.common_vnet_italy_north.resource_group_name
+  virtual_network_name                      = data.azurerm_virtual_network.common_vnet_italy_north.name
+  private_endpoint_network_policies_enabled = true
+
+  service_endpoints = [
+    "Microsoft.Web",
+    "Microsoft.AzureCosmosDB",
+    "Microsoft.Storage",
+  ]
+
+  delegation = {
+    name = "default"
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+}
+
+module "function_fast_login_itn" {
```

Comment:
Autoscale settings are missing

Reply:
Autoscale will be added with another PR https://github.com/pagopa/io-infra/pull/1198